src/WelcomeModal.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect} from 'react';
+import Modal from 'react-bootstrap/Modal';
+function WelcomeModal() {
+    const [show, setShow] = useState(false);
+
+  const handleClose = () => {
+    setShow(false);
+    localStorage.setItem('confessit.welcome', true)
+  }
+
+  useEffect(() => {
+    const hasSeenWelcome = localStorage.getItem('confessit.welcome')
+    if (!hasSeenWelcome){
+        setShow(true);
+    }
+  },[])
+
+  return (
+    <>
+      <Modal show={show} onHide={handleClose} size="xl">
+        <Modal.Header closeButton>
+          <Modal.Title>Welcome!</Modal.Title>
+        </Modal.Header>
+        <Modal.Body><span className="text-primary">ConfessIt</span> is a tool to help Roman Catholics walk through an examination of conscience prior to going to confession. 
+        We hope you'll find this useful to help remember sins you've committed since your last confession. 
+        Just check the <strong>Yes</strong> box next to sins in the Examine list, or tap the plus button to add your own. 
```

Comment:
A bit nit-picky, but let's make `Examine` strong too since it's the name of a column, like Review and Walkthrough.

Reply:
fixed